examples/acquiring_data/downloading_cutouts.py

```diff
@@ -78,37 +59,16 @@
 # We request one image every 2 hours.
 
 query = Fido.search(
-    a.Time(cutouts.date - 6*u.h, cutouts.date + 6*u.h),
-    a.Wavelength(cutouts.wavelength),
+    a.Time(amap.date - 6*u.h, amap.date + 6*u.h),
+    a.Wavelength(amap.wavelength),
     a.Sample(2*u.h),
     a.jsoc.Series.aia_lev1_euv_12s,
     a.jsoc.Notify(jsoc_email),
     a.jsoc.Segment.image,
     cutout,
 )
+print(query)
 
 #####################################################
-# Submit the export request and download the data.
-
-files = Fido.fetch(query)
-files.sort()
-
-#####################################################
-# Now that we've downloaded the files, we can create
-# a `~sunpy.map.MapSequence` from them.
-
-sequence = sunpy.map.Map(files, sequence=True)
```

Comment:
Why not also show download and load into a Map? It's all part of a single "task" in my opinion, you aren't going to want to search for and download stuff without wanting to load them after?